### DISCUSSION TITLE
TypeError: argument of type 'NoneType' is not iterable

### DISCUSSION THREAD

**Discussion by Henry-5** [openai/whisper#2055] [Q&A]
Hi, I am a beginner using Whisper.When I tried to use Whisper in PyCharm, an error occurred as follows:Traceback (most recent call last):
  File "G:\Bs\test.py", line 1, in <module>
    import whisper
  File "G:\Bs\venv\Lib\site-packages\whisper.py", line 69, in <module>
    libc = ctypes.CDLL(libc_name)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "G:\Anaconda\Lib\ctypes\__init__.py", line 366, in __init__
    if '/' in name or '\\' in name:
       ^^^^^^^^^^^
TypeError: argument of type 'NoneType' is not iterable

My environment configuration is：py3.11，torch2.0.0，whisper1.10,I tried reinstalling in a new virtual environment but failed. How can I solve this problem

**Comment by phineas-pta** [comment]
`openai-whisper` doesnt have v1.10

u installed the wrong package
